ansys/mapdl/core/launcher.py

```diff
@@ -146,8 +146,26 @@ def port_in_use(port, host=LOCALHOST):
             return True
 
 
+def delete_ip_file(path):
+    """Delete the created 'mylocal.ip'."""
+    # path=''
+    file_name = os.path.join(path, 'mylocal.ip') 
+    if os.path.exists(file_name):
+        # os.remove(file_name)
+        pass 
+
+
+def create_ip_file(ip, path):
+    """Create 'mylocal.ip' file required for ansys to change the IP of the gRPC server."""
+
+    file_name = os.path.join(path, 'mylocal.ip')
+    with open(file_name, 'w') as f:
+        f.write(f"{ip}")
```

Comment:
IP will always be a string, so there's no reason to use a f-string here.

```suggestion
        f.write(ip)
```

Reply:
Fixed in new commit.